aws/sdk/examples/s3/src/bin/put-object-presigned.rs

```diff
@@ -75,21 +75,50 @@ async fn main() -> Result<(), Box<dyn Error>> {
         .put_object()
         .bucket(&bucket)
         .key(&object)
+        .metadata("some-metadata", "abcd")
         .presigned(PresigningConfig::expires_in(expires_in)?)
         .await?;
-    println!("From client: {:?}", presigned_request.uri());
+    println!("From client: {:?}\n", presigned_request.uri());
 
     // Or, they can be made directly from an operation input
     let presigned_request = PutObjectInput::builder()
         .bucket(bucket)
         .key(object)
+        .metadata("some-metadata", "abcd")
         .build()?
         .presigned(
             &Config::from(&shared_config),
             PresigningConfig::expires_in(expires_in)?,
         )
         .await?;
-    println!("From operation input: {:?}", presigned_request.uri());
+    println!("From operation input: {:?}\n", presigned_request.uri());
+
+    print_as_curl_request(&presigned_request, Some("some test content"));
```

Comment:
we should probably also demonstrate the conversion to `http::Request` that you just added